ReactSkia/components/RSkComponentText.cpp

```diff
@@ -16,12 +16,14 @@ namespace react {
 RSkComponentText::RSkComponentText(const ShadowView &shadowView)
     : RSkComponent(shadowView) {}
 
+void RSkComponentText::updateComponetProps(const ShadowView &newShadowView,bool forceUpadte) {}
```

Comment:
If there is no implementation for this function , then would suggest this be done in header file itself. Than having it at two places.
Same applies for RSkComponentRawText::updateComponetProps()

Reply:
As a reference onPaint, They wrote declaration in header file and empty definition in .cpp.